checks/evaluation/ci_tests.go

```diff
@@ -128,7 +128,7 @@ func isTest(s string) bool {
 	for _, pattern := range []string{
 		"appveyor", "buildkite", "circleci", "e2e", "github-actions", "jenkins",
 		"mergeable", "packit-as-a-service", "semaphoreci", "test", "travis-ci",
-		"flutter-dashboard", "Cirrus CI",
```

Comment:
It is essential to maintain consistency by preserving the existing entry, as other projects may rely on this.

Reply:
That's fair, although Cirrus CI support is current broken because the string it is compared to below has already been converted to lowercase, and this entry has uppercase letters in it.

I verified it by adding a unit test locally that had both Cirrus CI and cirrus ci in it, and the tests failed. I don't have access to Cirrus CI so I am unable to verify the correct string to use.

I recognize it might make sense to leave it in there though, so it can be fixed properly instead of removing it, so I'm bringing it back for now.